src/components/Square.js

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+const Square = ({ image, idx, fabric, debug, quiltSectionWidth }) => {
+  // if even number wide, true.
+  // if not even number wide, make even every other row.
+  //   const isEven =
+  //     quiltSectionWidth % 2 === 0 ||
+  //     Math.floor(idx / quiltSectionWidth) % 2 === 0;
+  return (
+    <div
+      css={[
+        {
+          overflow: "hidden",
+          position: "relative",
+          clipPath: "polygon(0% 0%, 100% 0%, 100% 100%, 0% 100%)",
+          width: `calc(125vw / ${quiltSectionWidth})`,
+          height: `calc((8 * (125vw / ${quiltSectionWidth})) / 9)`,
+          margin: 0,
+          backgroundRepeat: "repeat",
+          backgroundImage: `url(./images/${fabric}/${fabric}${image}.jpg)`,
+          backgroundPosition: "center",
+          marginTop: `calc((8 * 125vw / ${quiltSectionWidth}) / 9 / 2)`
```

Comment:
Because we're not needing to offset every other shape, I don't think we need this marginTop at all. 

```suggestion
```